conferences/2022/security.json

```diff
@@ -464,5 +464,17 @@
     "cfpEndDate": "2022-08-20",
     "twitter": "@avar_asia",
     "url": "https://aavar.org/cybersecurity-conference"
+  },
+  {
+    "name": "Qubit Conference Tatry",
+    "startDate": "2022-11-09",
+    "endDate": "2022-11-10",
+    "city": "Demänovská Dolina",
```

Comment:
```suggestion
    "city": "Tatras",
```

Reply:
Looks like the name of the city is Tatras 
<img width="333" alt="image" src="https://user-images.githubusercontent.com/445045/181776068-97d23637-1d95-4cf4-b33b-cc6d1f93a743.png">